IBPSA/BoundaryConditions/BESTEST.mo

```diff
@@ -0,0 +1,1718 @@
+<li>Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2)</li>
+</ul>
+<p><br><b>Hourly Outputs:</b> The following outputs shall be provided for each hour of the days specified for each test case in Table 3:</p>
+<ul>
+<li>Dry bulb temperature (&deg;C)</li>
+<li>Relative humidity (&percnt;)</li>
+<li>Dewpoint temperature (&deg;C)</li>
+<li>Humidity ratio (kg moisture/kg dry air)</li>
+<li>Wet bulb temperature (&deg;C)</li>
+<li>Windspeed (m/s)</li>
+<li>Wind direction (degrees from north)</li>
+<li>Station pressure (mbar)</li>
+<li>Total cloud cover (tenths of sky)</li>
+<li>Opaque cloud cover (tenths of sky)</li>
+<li>Sky temperature (&deg;C)</li>
+</ul>
+<p><br>&bull; Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2)&nbsp;</p>
+<p><br><i>Table 3: Specific Days for Output</i></p>
+<table cellspacing=\"2\" cellpadding=\"0\" border=\"1\"><tr>
+<td><p>Case </p></td>
+<td><p>Days</p></td>
+</tr>
+<tr>
+<td><p>WD100 </p></td>
+<td><p>May 4th, July 14th, September 6th</p></td>
+</tr>
+<tr>
+<td><p>WD200 </p></td>
+<td><p>May 24th, August 26th</p></td>
+</tr>
+<tr>
+<td><p>WD300 </p></td>
+<td><p>February 7th, August 13th</p></td>
+</tr>
+<tr>
+<td><p>WD400 </p></td>
+<td><p>January 24th, July 1st</p></td>
+</tr>
+<tr>
+<td><p>WD500 </p></td>
+<td><p>March 1st, September 14th</p></td>
+</tr>
+<tr>
+<td><p>WD600 </p></td>
+<td><p>May 4th, July 14th, September 6th</p></td>
+</tr>
+</table>
+<p><br><br><b>Sub-hourly Outputs:</b></p>
+<p><br>The following outputs shall be provided at each timestep of the days specified for each test case in Table 3:</p>
+<ul>
+<li>Dry bulb temperature (C)</li>
+<li>Relative humidity (&percnt;)</li>
+<li>Sum of total, beam, and diffuse solar radiation incident on each surface (Wh/m2) </li>
+</ul>
+<p><br>The following outputs shall be provided integrated hourly for the days specified for each test case in Table 3:</p>
+<ul>
+<li>Total incident horizontal solar radiation (Wh/m2)</li>
+<li>Total incident horizontal beam solar radiation (Wh/m2)</li>
+<li>Total incident horizontal diffuse solar radiation (Wh/m2) </li>
+</ul>
+<h4>Validation results</h4>
+<p>(Not available yet)</p>
+<h4>Implementation</h4>
+<p>To generate the data shown in this user guide, run </p>
+<p><span style=\"font-family: Courier New;\">cd IBPSA/Resources/src/BoundaryConditions/Validation/BESTEST</span></p>
+<p><span style=\"font-family: Courier New;\">python3 WeatherBESTEST.py</span></p>
+<h4>References</h4>
+<p>(Not available yet)</p>
+</html>"));
+  end UsersGuide;
+
+  package Validation
+    "Boundary conditions validation according to BESTEST specifications"
+    model WD100 "Test model for BESTEST weather data: base case"
+      extends Modelica.Icons.Example;
+      parameter Modelica.SIunits.Angle lat=0.6952170009469 "Latitude angle";
+      parameter Real rho=0 "Ground reflectance";
+
+      WeatherData.ReaderTMY3                          weaDat(
```

Comment:
Fix white space.

Reply:
I fixed the white spaces , sometimes though after I already fixed them, by saving the package again the spacing will become weird again. It also happens with the documentation

Reply:
In dymola I suggest to simply save the file but not the whole package to avoid that the formatting gets changed again.